static/_headers

```diff
@@ -3,3 +3,5 @@
  X-XSS-Protection: 1; mode=block
  Referrer-Policy: no-referrer
  X-Content-Type-Options: nosniff
+ Content-Security-Policy: default-src 'none'; object-src 'none'; img-src 'self'; script-src 'self'; style-src 'self'; font-src 'self';
```

Comment:
The changes to the _headers file don't appear to be related to the PR and should be omitted, unless I'm mistaken

Reply:
Yes. _headers file is here mistakenly.